### PR TITLE
T14014

### PR DESCRIPTION
**Pull request by tchx84**
https://phabricator.endlessm.com/T14014

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/make -f
+
+%:
+	dh $@ --builddirectory=_build
+
+override_dh_auto_configure:
+	dh_auto_configure -- \
+	--prefix=${DEB_PREFIX} \
```
Comment (ptomato) [inline debian/rules:8]: Isn't all this done automatically?

File: debian/source/format [hunk 2]
```diff
@@ -0,0 +1 @@
+3.0 (native)
```
Comment (ptomato) [inline debian/source/format]: Should be `quilt`, I believe, `native` is only for tools written by Debian themselves

File: debian/control [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+Architecture: any
+Depends: ${misc:Depends},
+         libemeus-1.0-1 (= ${binary:Version})
+Description: Emeus headers and development files
+ Emeus is a constraint-based layout manager for GTK+.
+
+Package: gir1.2-emeus-1.0
+Section: introspection
+Architecture: any
+Depends: ${misc:Depends},
```
Comment (ptomato) [inline debian/control]: Needs `${gir:Depends}` here as well

File: debian/control [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+Maintainer: Martin Abente Lahaye <martin@endlessm.com>
+Build-Depends: autotools-dev,
+               debhelper (>= 8.0.0),
+               gir1.2-glib-2.0,
+               gtk-doc-tools (>= 1.18),
+               gobject-introspection,
+               libgirepository1.0-dev (>= 1.30),
+               libglib2.0-dev (>= 2.38),
+               libgtk-3-dev,
+               meson,
```
Comment (ptomato) [inline debian/control]: `meson (>= 0.36)`?

File: debian/control [hunk 5]
```diff
@@ -0,0 +1,49 @@
+Source: emeus
+Priority: standard
+Maintainer: Martin Abente Lahaye <martin@endlessm.com>
+Build-Depends: autotools-dev,
```
Comment (ptomato) [inline debian/control]: What does `autotools-dev` provide here?

**Review by ptomato — CHANGES_REQUESTED** [review]
Some minor comments, also questions that I'm not sure about.

Just FYI, we decided to avoid separate debian branches on new repositories if possible, but in this case it's not possible since we're not the upstream.

File: debian/rules (hunk 1 above)
Comment (tchx84) [reply debian/rules:8]: I found two issues which I haven't found other ways to solve: a) make it use multi-arch path for libdir and b) dh_auto_configure receives paths without substitution:

```
./configure --build=x86_64-linux-gnu --prefix=/usr --includedir=\${prefix}/include --mandir=\${prefix}/share/man --infodir=\${prefix}/share/info --sysconfdir=/etc --localstatedir=/var --disable-silent-rules --libexecdir=\${prefix}/lib/emeus --disable-maintainer-mode --disable-dependency-tracking
```

I don't know really if we should give special treatment to those in our configure file.

**Comment by tchx84** [comment]
@ptomato pushed the changes, but as we talked, I left the params in the rules so we don't need to make that configure script a real configure file.

**Review by ptomato — APPROVED** [review]

**Pull request merged**